tests/phpunit/api/v3/OptionValueCHTest.php

```diff
@@ -117,7 +124,9 @@ public function tearDown() {
       $this->callAPISuccess('CustomField', 'delete', ['id' => $this->customField['id']]);
       $this->callAPISuccess('CustomGroup', 'delete', ['id' => $this->customGroup['id']]);
     }
-    $this->callAPISuccess('FinancialType', 'delete', ['id' => $this->fund['id']]);
+    $this->callAPISuccess('FinancialType', 'get', ['id' => $this->fund['id'], 'api.FinancialType.delete' => '"id":"$value.id"']);
+    $this->callAPISuccess('Contact', 'delete', ['id' => $this->individualID]);
+    CRM_Core_DAO::executeQuery("DELETE FROM civicrm_ch_contribution_batch ");
```

Comment:
What is the context in which our tests run? This seems particularly destructive of test environments.